cardpresent/src/main/java/com/fattmerchant/android/anywherecommerce/AwcUtils.kt

```diff
@@ -93,4 +98,8 @@ internal fun TransactionResult.Companion.from(transaction: AnyPayTransaction): T
         TransactionType.VOID -> "void"
         else -> "sale"
     }
+}
+
+internal fun com.fattmerchant.omni.data.models.Transaction.awcExternalId(): String? {
+    return (meta as? Map<*, *>)?.get("awcTransactionId") as? String
```

Comment:
:(

Reply:
@HassanNazari fixed in b8bc7a4 :) Nice catch!